Extension/src/Debugger/attachToProcess.ts

```diff
@@ -103,15 +103,29 @@ export class RemoteAttachPicker {
         });
     }
 
+    // Creates a string to run on the host machine which will execute a shell script on the remote machine to retrieve OS and processes
+    private getRemoteProcessCommand(): string {
+        let shellCommandQuote: string = `'`;
+        let hostCommandQuote: string = `"`;
+
+        // Must use single quotes around the whole command and double quotes for the argument to `sh -c` because Linux evaluates $() inside of double quotes.
```

Comment:
Can you call it outer and inner quote? isn't the whole thing also a shell command?